lib/styles/settings/_colors.scss

```diff
@@ -1,89 +1,82 @@
-// regular colors
-$color-elf-green: #239952;
-$color-garden-green: #98d293;
-$color-ice-green: #e2f2e3;
-$color-jungle-green: #27aa5b;
-$color-mantis-green: #8bc34a;
-$color-salad-green: #4ab24f;
-$color-wasabi-green: #aed581;
-
-$color-crimson-red: #de2f1b;
-$color-french-red: #e55066;
-$color-brush-red: #ee978d;
-
-$color-cobalt-blue: #2f4999;
-$color-navy-blue: #0277bd;
-$color-baby-blue: #81d4fa;
-
-$color-ice-yellow: #fef3dd;
-$color-lemon-yellow: #fff176;
-$color-gorse-yellow: #fdd835;
-$color-mustard-yellow: #ffd54f;
-$color-amber-yellow: #ffc107;
-$color-goldfish-yellow: #fdb72d;
-$color-california-yellow: #eca005;
-
-$color-melon-orange: #ff8a65;
-$color-outrageous-orange: #ff5722;
-$color-satin-orange: #cc3e17;
-$color-ice-orange: #ffe6ca;
-
-$color-gigas-violet: #62378d;
-$color-deluge-violet: #7b569f;
-
-$color-total-black: #000;
-$color-firefly-black: #0c1726;
-$color-catalina-blue: #25335a;
-
-$color-powder-pink: #fadeda;
-
-$color-rhino-gray: #434a5a;
-$color-storm-gray: #6f7285;
-$color-cadet-gray: #a7acbd;
-$color-mischka-gray: #cdd0da;
-$color-athens-gray: #e6e8ee;
-$color-ice-gray: #eff0f3;
-$color-alabaster-gray: #f6f6f8;
-
-$color-total-white: #fff;
-
-$color-primary-rgb: var(--color-primary-rgb);
-$color-primary-supporting-rgb: var(--color-primary-supporting-rgb);
-$color-primary-opacity25: rgba(var(--color-primary-rgb), 0.25);
-$color-secondary-rgb: var(--color-secondary-rgb);
-$color-secondary-supporting-rgb: var(--color-secondary-supporting-rgb);
-$color-total-white-opacity50: rgba(255, 255, 255, 0.5);
+@import '../../styles/settings/colors/raw';
+
+$color-firefly-black: var(--raw-black);
+$color-firefly-black-rgb: var(--raw-black-rgb);
+$color-total-white: var(--raw-white);
+
+$color-wasabi-green: var(--raw-green-200);
```

Comment:
what about these colors? Didn't we agreed to revert them until stage 4?

Reply:
Fixed, I edded removed unused colours.

Reply:
I'm not sure what's fixed. I still see these colors on a diff in PR, so they are changed. Why?

Reply:
You mean I should revert:
```
$color-wasabi-green: var(--raw-green-200);
```
to:
```
$color-wasabi-green: #aed581;
```
? I don't see any point in taking it back

Reply:
We talked about it at least 2 times already. We agreed that we move this change into stage 4. I don't undestand why you push on this now.

This change isn't backwards compatible. You can't pass CSS variables into `mix(), darken(), lighten()` SASS functions. Example: https://github.com/bethinkpl/wnl-platform/blob/9e34e6e69bec18340297af2c35b665f969108a89/resources/assets/js/components/global/Sidenav.vue#L33

Reply:
Right. File reverted @kvas-damian